auth/pom.xml

```diff
@@ -91,6 +95,17 @@
             <artifactId>jsr305</artifactId>
             <version>3.0.2</version>
         </dependency>
+        <dependency>
+			<groupId>org.springframework</groupId>
```

Comment:
Can we fix the formatting here please?

Reply:
Do you guys use any specific formatter? I remember it happened in my last PR too.